api/internal/core/entity/entity.go

```diff
@@ -81,7 +81,7 @@ type Route struct {
 	Hosts           []string               `json:"hosts,omitempty"`
 	RemoteAddr      string                 `json:"remote_addr,omitempty"`
 	RemoteAddrs     []string               `json:"remote_addrs,omitempty"`
-	Vars            interface{}            `json:"vars,omitempty"`
+	Vars            []interface{}          `json:"vars,omitempty"`
```

Comment:
Why should the data format be modified here? Whether it has influence on the existing FE and BE logic.

Reply:
The same question as @Jaycean.

Reply:
@Jaycean  please take a look at `Bugfix Description`, I wrote the reason there.

Reply:
Sorry, my question. I didn't read the description carefully.

Reply:
cc @nic-chen 
FE CI failed:https://github.com/apache/apisix-dashboard/pull/1594/checks?check_run_id=2110905208
Remove the corresponding data: `x-apimix-vars ": []`